src/ralph/models/edx/video/statements.py

```diff
@@ -0,0 +1,191 @@
+"""Video event model definitions"""
+
+from typing import Literal, Union
+
+from pydantic import Json
+
+from ralph.models.edx.video.fields.events import (
+    HideTranscriptEventField,
+    PauseVideoEventField,
+    PlayVideoEventField,
+    SeekVideoEventField,
+    ShowTranscriptEventField,
+    SpeedChangeVideoEventField,
+    StopVideoEventField,
+    VideoBaseEventField,
+)
+from ralph.models.selector import selector
+
+from ..browser import BaseBrowserModel
+
+
+class UILoadVideo(BaseBrowserModel):
+    """Represents the `load_video` event model.
+
+    Attributes:
+        event (VideoBaseEventField): See VideoBaseEventField.
+        event_type (str): Consists of the value `load_video`.
+    """
+
+    __selector__ = selector(event_source="browser", event_type="load_video")
+
+    event: Union[
+        Json[VideoBaseEventField],  # pylint: disable=unsubscriptable-object
+        VideoBaseEventField,
+    ]
+    event_type: Literal["load_video"]
```

Comment:
I think we could. There was also this issue for previous events we modelized but it would be the first time that we allow two names. Maybe should we keep in mind to modify in the other events in a future pull request?

Reply:
Ok, sure) this can be done in a future pull request) Let's keep then the `name` field equal to the `event_type`)

Apropos, we have a couple of video events for validation on [pastebin](https://pastebin.com/KycKt0uf))
Could you check whether the command `cat ../edx_video.json | ./bin/ralph validate -f edx` passes without failure?

Reply:
That's a good point! It seems to me that all browser (UI) edX events (in the FUN version of edX dogwood) have a `name` field) and for the video events, the name field is equal to the `event_type` field. It might also be true that the value is `edx.video.paused` for the mobile app and that the name field was removed in the newer versions of edX. I think it's worth exploring)
I guess, it depends on whether we want to support one video event from the dogwood version and future versions of edX with a single pydantic class or multiple ones.

Reply:
The aim is to support video events for everyone but us as we do not use this events. So maybe, should we consider all the possibilities ? Add this field as optional and with the two possible values ? What do you think ?

Reply:
I like this idea) :+1:

Reply:
Okay, let's go! :muscle: